uberfire-rest/uberfire-rest-backend/src/main/java/org/guvnor/rest/backend/JobRequestHelper.java

```diff
@@ -436,23 +425,23 @@ public JobResult deployProject(final String jobId,
         }
     }
 
-    public JobResult removeOrganizationalUnit(final String jobId,
-                                              final String organizationalUnitName) {
+    public JobResult removeSpace(final String jobId,
+                                 final String spaceName) {
         JobResult result = new JobResult();
         result.setJobId(jobId);
 
-        if (organizationalUnitName == null) {
+        if (spaceName == null) {
             result.setStatus(JobStatus.BAD_REQUEST);
             result.setResult("OrganizationalUnit name must be provided");
```

Comment:
I think that it would be better if the result message used "Space" instead of "OrganizationalUnit". What do you think?